net-misc/htbinit: fix LICENSE, EAPI8 bump

mm1ke opened the pull request:
Hi,

This updates `net-misc/htbinit` for `EAPI8`.
With `EAPI8` i've decided to remove the `ipv6` flag since it only adds a small patch.

ulm commented:
Either drop `r8` to unstable (which I'd prefer), or keep it stable and drop `r7`.
Also, I don't see EAPI 8.

mm1ke commented:
> Either drop `r8` to unstable (which I'd prefer), or keep it stable and drop `r7`. Also, I don't see EAPI 8.

Thx, fixed it. Dropped `r8` to unstable and add `EAPI8`.
